extension/manifest.json

```diff
@@ -75,7 +75,7 @@
       "js": ["shared.js","keysocket-grooveshark.js"]
     },
     {
-      "matches": ["https://www.amazon.com/gp/dmusic/mp3/player*"],
+      "matches": ["https://www.amazon.com/gp/dmusic/cloudplayer/web*"],
```

Comment:
I synced from git but forgot to switch branches. Now with the correct branch and the extension reloaded I do see the icon in the URL path bar, but button presses still do not work.